### PR TITLE
snoop: add Hindi translation

### PR DESCRIPTION
**Pull request by karthik-script**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

https://github.com/tldr-pages/tldr/issues/13571

I will add more translations in subsequent PRs

### CODE REVIEW

**Review by Magrid0 — COMMENTED** [review]
LGTM after the above suggestions

File: pages.hi/sunos/snoop.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`snoop -o {{path/to/file}}`
+
+- एक फ़ाइल से पैकेट का विस्तृत प्रोटोकॉल स्तर सारांश प्रदर्शित करें:
+
+`snoop -V -i {{path/to/file}}`
+
+- एक होस्टनाम से आने वाले और एक दिए गए पोर्ट पर जाने वाले नेटवर्क पैकेट कैप्चर करें:
+
+`snoop to port {{port}} from host {{hostname}}`
```
Comment (kbdharun) [inline pages.hi/sunos/snoop.md:21]:
```suggestion
`snoop -V -i {{फ़ाइल/का/पथ}}`

- एक होस्टनाम से आने वाले और एक दिए गए पोर्ट पर जाने वाले नेटवर्क पैकेट कैप्चर करें:

`snoop to port {{पोर्ट}} from host {{होस्टनाम}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by debghs — APPROVED** [review]
LGTM.

**Pull request merged**